releasenotes/notes/oracle-data-guard-28e4f91823d95cbb.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Breaking change for Data Guard Monitoring: Renaming Data Guard metrics `oracle.apply_lag` and `oracle.transport_lag` to `oracle.data_guard.apply_lag` and `oracle.data_guard.transport_lag`.
```

Comment:
Changelog entries are not markdown, but RST and use double backticks for monospace.

Reply:
ack